build/Targets/References.targets

```diff
@@ -117,7 +117,10 @@
 		if it is not provided here, tests fail to load this assembly at runtime.
 	-->
         <Reference Include="Microsoft.VisualStudio.Diagnostics.PerformanceProvider">
-          <HintPath>$(DevEnvDir)\PrivateAssemblies\Microsoft.VisualStudio.Diagnostics.PerformanceProvider.dll</HintPath>
+          <HintPath Condition="Exists('$(DevEnvDir)\PrivateAssemblies\Microsoft.VisualStudio.Diagnostics.PerformanceProvider.dll')">$(DevEnvDir)\PrivateAssemblies\Microsoft.VisualStudio.Diagnostics.PerformanceProvider.dll</HintPath>
```

Comment:
I see your point.  I would prefer if this was never checked in and we just used package refs instead but if that is not viable then I'll move this logic into the calculation of `DevEnvDir` in a mop-up PR